polymorphic/compat.py

```diff
@@ -0,0 +1,10 @@
+import django
+
+DJANGO_GTE_2 = django.VERSION >= (2, 0)
+
+try:
+    # Django 2.0+
+    from django.urls import URLResolver
+except ImportError:
+    # Django < 2.0
+    from django.urls import RegexURLResolver as URLResolver
```

Comment:
There is only a single place where this import is needed. Does that quality for a separate file to do this?